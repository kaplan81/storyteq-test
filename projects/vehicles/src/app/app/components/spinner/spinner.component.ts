import { ChangeDetectionStrategy, Component } from '@angular/core';

@Component({
  changeDetection: ChangeDetectionStrategy.OnPush,
  selector: 'teq-spinner',
  standalone: true,
  imports: [],
  templateUrl: 'spinner.component.html',
  styleUrl: 'spinner.component.scss',
})
export class SpinnerComponent {}
