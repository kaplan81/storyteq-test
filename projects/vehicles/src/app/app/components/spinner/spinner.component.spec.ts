import { ComponentFixture, TestBed } from '@angular/core/testing';
import { SpinnerComponent } from './spinner.component';

describe('SpinnerComponent', () => {
  let fixture: ComponentFixture<SpinnerComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [SpinnerComponent],
    }).compileComponents();
    fixture = TestBed.createComponent(SpinnerComponent);
  });

  it('should match snapshot', () => {
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });
});
