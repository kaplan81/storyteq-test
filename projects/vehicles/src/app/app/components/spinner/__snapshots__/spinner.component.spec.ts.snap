// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerComponent should match snapshot 1`] = `
<teq-spinner>
  <span
    class="spinner"
  />
</teq-spinner>
`;
